From d1f61050d5723681cd27da60d86885618003f952 Mon Sep 17 00:00:00 2001
From: Thao Nguyen <thao.nguyen.yb@renesas.com>
Date: Sat, 5 Aug 2017 12:39:02 +0700
Subject: [PATCH] WORKAROUND: using GLES3/gl3ext.h from mesa to prevent error
 of gstreamer1.0-plugins-bad

According to the Khronos OpenGL ES Registry, the extension header for
GLES 3.0 is actually <GLES2/gl2ext.h>.
gl3ext.h should be empty and provided only for legacy compatibility.

Signed-off-by: Thao Nguyen <thao.nguyen.yb@renesas.com>
Signed-off-by: Thuy Tran <thuy.tran.xh@rvc.renesas.com>
---
 usr/include/GLES3/gl3ext.h | 24 ++++++++++++++++++++++++
 1 file changed, 24 insertions(+)
 create mode 100644 usr/include/GLES3/gl3ext.h

diff --git a/usr/include/GLES3/gl3ext.h b/usr/include/GLES3/gl3ext.h
new file mode 100644
index 0000000..4d4ea96
--- /dev/null
+++ b/usr/include/GLES3/gl3ext.h
@@ -0,0 +1,24 @@
+#ifndef __gl3ext_h_
+#define __gl3ext_h_
+
+#include <GLES2/gl2ext.h>
+/* $Revision: 17809 $ on $Date:: 2012-05-14 08:03:36 -0700 #$ */
+
+/*
+ * This document is licensed under the SGI Free Software B License Version
+ * 2.0. For details, see http://oss.sgi.com/projects/FreeB/ .
+ */
+
+/* OpenGL ES 3 Extensions
+ *
+ * After an OES extension's interactions with OpenGl ES 3.0 have been documented,
+ * its tokens and function definitions should be added to this file in a manner
+ * that does not conflict with gl2ext.h or gl3.h.
+ *
+ * Tokens and function definitions for extensions that have become standard
+ * features in OpenGL ES 3.0 will not be added to this file.
+ *
+ * Applications using OpenGL-ES-2-only extensions should include gl2ext.h
+ */
+
+#endif /* __gl3ext_h_ */
+
-- 
1.9.1

